Alignment/LaserDQM/plugins/LaserDQM.h

```diff
@@ -15,12 +15,13 @@
 
 // DQM
 #include "DQMServices/Core/interface/DQMStore.h"
-#include "DQMServices/Core/interface/MonitorElement.h"
 
 #include <iostream>
 
 class LaserDQM : public edm::EDAnalyzer {
 public:
+  typedef dqm::legacy::DQMStore DQMStore;
+  typedef dqm::legacy::MonitorElement MonitorElement;
```

Comment:
Something to keep in mind for the future. We highly encourage the use of ```using``` instead of ```typedef``` in all future code development.

Reply:
I used `using` in some places, but then switched to `typedef` for symmetry with those cases where the declarations are put in namespaces (fewer, but still some).

Reply:
@schneiml could you please clarify the rationale behind this choice? Is there a specific reason I am missing, or is it mostly a matter of taste?

Reply:
Regarding typedef vs. using: `using` only works in class scopes, but some of these declarations also appear in namespaces, where afaik `using` can't be used. So I chose to use the form of declaration that works everywhere. 

Another option is to not import the names but just use full names. I also did that in a few places where the declarations caused trouble (e.g. due to multi-inheritance), but in general I preferred to have the import since it leads to less changes.

Reply:
`using` can also be used outside of class scope.